test/Dockerfile

```diff
@@ -0,0 +1,9 @@
+ARG ALPINE_VERSION="3.15"
+
+FROM docker.io/library/alpine:${ALPINE_VERSION}
+
+RUN apk add bash curl
+
+RUN curl -sSL https://apk.cloudposse.com/install.sh | bash
+
+RUN apk add atmos@cloudposse terraform@cloudposse
```

Comment:
Should we pin these packages ?

Reply:
updated in https://github.com/cloudposse/github-action-ci-docker/pull/1/commits/7b1ea68c18869725cdf5d6001d892b7193abb214